src/ui/public/index_patterns/_calculate_indices.js

```diff
@@ -2,6 +2,12 @@ define(function (require) {
   const _ = require('lodash');
   const moment = require('moment');
 
+  // gets parsed value if given arg is a moment object
+  function timeValue(val) {
+    const hasValueOfFn = _.isFunction(_.get(val, 'valueOf'));
```

Comment:
Actually, the goal here was really just to check for whether the object was a moment object.  I'd rather be overly explicit for that type than deal with the ramifications of supporting any possible `valueOf`, so I'm going to switch this to use `Moment.isMoment()` instead.